Library/Formula/menhir.rb

```diff
@@ -13,7 +13,10 @@ class Menhir < Formula
   depends_on "ocaml"
 
   def install
-    ENV.deparallelize
+    # Workaround parallelized build failure by running stage2 on its own.
+    # Submitted to menhir-list@yquem.inria.fr on 24th Feb 2016.
+    system "make -C src library stage1"
```

Comment:
Thank you for explaining! I knew there must be some reason other than commaphilia.